.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: NAME-EXAMINATION UI CI
+
+on:
+  pull_request:
+    types: [assigned, synchronize]
+    paths:
+        - "web/**"
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./web
+
+jobs:
+  setup-job:
+    runs-on: ubuntu-20.04
+
+    if: github.repository == 'bcgov/name-examination'
+
+    steps:
+      - uses: actions/checkout@v3
+      - run: "true"
+
+  linting:
+    needs: setup-job
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        node-version: [18.16.0]
```

Comment:
Are you sure you want to use a Node version different from everyone else? It would mean some people will have to use nvm (node version manager).

Reply:
I'm using nvm as well, I figured if we're modernizing the project we should use the latest version for node as well

Reply:
I don't disagree but I'd want to weigh the pros and cons.

Reply:
Yes, we should use the same version (16.14.2) as other applications. We haven't review the impact on the new version of node. There will be a new ticket to upgrade Node when we complete the VUE3 upgrade.

Reply:
Ok, will use 16.14.2 for this project.